designs/developer/add-45/sections/pipelines/pipelines.md

```diff
@@ -0,0 +1,83 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Pipelines section in Add Flows (added to 4.5)
+* Updates to the Pipelines section of the 'Add' page include moving the Pipelines section below the Resources section. In addition, the ‘Show pipeline visualization’ component will remain open even if the user selects a new builder image or resource during their session.
+
+## Import from Git
+### No Builder Image Selected
+* When no builder image or resource has been selected yet, the Pipelines section will display an info alert that says “Select a builder image and resource to see if there is a pipeline template available for this runtime.”
+
+![image](img/info-alert-1.png)
+
+### Builder Image Selected and Pipeline Template Exists
+* When a builder image has been selected, if a pipeline template is available, the Pipeline section shall be updated to display an “Add pipeline”  checkbox option.
+* The default state of the Add pipeline checkbox is OFF.
+* An expandable control with text “Show pipeline visualization” is displayed below the Add pipeline checkbox and is collapsed by default.
+* A 'Dev Preview' label is featured next to 'Add pipeline' label.
+
+![Add pipeline](img/import-from-git-1.png)
+
+* After the user clicks the caret or 'Show pipeline visualization' label, the visualization is expanded and the user can hover over the different sections of the pipeline to see more details.  
+* The pipeline visualization label changes to ‘Hide pipeline visualization’ if the pipeline visualization is expanded.
+
+![image](img/import-from-git-2.png)
+
+
+### Builder Image Selected and No Pipeline Template Exists
+* If a selected combination of a builder image and resource does not support a pipeline an info alert would appear alerting the user of this.
+* This message specifies the builder image and resource type that the user selected.
+
+![image](img/info-alert-2.png)
+
+
+### Developer Catalog
+
+#### Pipeline Template Exists
+* When an app, service, or source-to-image builder is added, if a pipeline template is available, the Pipeline section is updated to display an “Add pipeline”  checkbox option
+* The default state of the Add pipeline checkbox is OFF
+* An expandable control with text “Show pipeline visualization” is displayed below the Add pipeline checkbox and is collapsed by default
+
+![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit.png)
```

Comment:
@lwrigh Do all these examples need the "dev preview" label?

Reply:
@serenamarie125 Is that accurate? From more recent changes I would assume they do.